extensions/amp-sidebar/0.2/amp-sidebar.css

```diff
@@ -80,6 +80,10 @@ amp-sidebar[side], .i-amphtml-sidebar-mask {
   margin: 0;
 }
 
+.amphtml-sidebar-mask {
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
 .i-amphtml-sidebar-mask {
```

Comment:
Same here 
```suggestion
.amp-sidebar-mask {
  background-color: rgba(0, 0, 0, 0.5);
```

Reply:
Keeping both names to prevent a larger scope.